apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainCheckerSamples.kt

```diff
@@ -27,4 +27,28 @@ class IterableLikeToContainCheckerSamples {
             }
         }
     }
+
+    @Test
+    fun notOrAtMost() {
+        expect(listOf("A", "A", "B", "C")).toContain.inAnyOrder.notOrAtMost(2).entry {
+            toEqual("A")
+        }
+
+        expect(listOf(1, 2, 3)).toContain.inAnyOrder.notOrAtMost(2).entry {
+            toBeGreaterThan(1)
+        }
+
+        fails {
```

Comment:
would be nice if you explain why it fails (same same below)
```suggestion
        fails { // because "B" is 3 times in the List
```